tests/unit/JobStatusTest.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\Yii\Queue\Tests\unit;
+
+use Yiisoft\Yii\Queue\Enum\JobStatus;
+use Yiisoft\Yii\Queue\Tests\TestCase;
```

Comment:
Now it's useless. It should be replaced to `PHPUnit\Framework\TestCase`
```suggestion
use PHPUnit\Framework\TestCase;
```

Reply:
There is container rebuilding in the base test. It's not useless.